.sync.yml

```diff
@@ -1,4 +1,7 @@
 ---
+.gitattributes:
+  include:
+    '.gitattributes/*': 'eol=lf'
```

Comment:
how does this create `files/shell/* eol=lf` above?

Reply:
oops

Reply:
fixed